src/util.py

```diff
@@ -74,21 +228,27 @@ def truncate_colormap(cmap, minval=0.0, maxval=1.0, n=100, name=None):
     cm.register_cmap(name, new_cmap)
     return new_cmap
 
-################################################
-
-def make_byr_cmap():
+def xr_add_cyclic_longitudes(da, coord):
     """
-    Define the byr colormap
+    Utility function to handle the no-shown-data artifact of 0 and 360-degree longitudes
+
+    Args:
 
-    Note: this will be replaced with cmaps.BlueYellowRed
+        da (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains one or more coordinates, strictly including the coordinate with the name
+            given with the "coord" parameter.
+
+        coord (:class:`str`):
+            Name of the longitude coordinate within "da" data array.
     """
-    from . import cmaps
 
-    import warnings
-    warnings.simplefilter('always', DeprecationWarning)
-    warnings.warn('geocat.viz.util.make_byr_cmap: This function has been '
-                  'deprecated, please use geocat.viz.cmaps.BlueYellowRed '
-                  'instead.', DeprecationWarning, stacklevel=2)
-    warnings.filters.pop(0)
+    import xarray as xr
+    import cartopy.util as cutil
+
+    cyclic_data, cyclic_coord = cutil.add_cyclic_point(da.values, coord=da[coord])
+
+    coords = da.coords.to_dataset()
+    coords[coord] = cyclic_coord
+
+    return xr.DataArray(cyclic_data, dims=da.dims, coords=coords.coords, attrs=da.attrs, encoding=da.encoding)
```

Comment:
A quick heads up, I found an issue with this `encoding` keyword argument being deprecated in Xarray as of version 0.14 and removed in 0.15. See NCAR/GeoCAT-examples#55 for a possible fix.